lecli/query/commands.py

```diff
@@ -11,14 +11,12 @@
 @click.command()
 # nargs (-1) makes sure multiple log keys is supported
 @click.argument('logkeys', type=click.STRING, nargs=-1)
-@click.option('-n', '--lognick', default=None,
-              help='Nickname of log in config file')
-@click.option('-g', '--loggroup', default=None,
-              help='Name of log group defined in config file')
+@click.option('-c', '--favorites', default=None,
```

Comment:
-c? I know -f is gone but this is not very intuitive.

Reply:
yea running out of letters, we're not going to get the most intuative - although in the case (at a reach ) 'c' for CLI Favorites?????